server/channels/issue.js

```diff
@@ -0,0 +1,62 @@
+const broadcast = require('../utils').broadcast;
+const emit = require('../utils').emit;
+const logger = require('../logging');
+
+const addQuestion = require('../models/issue').addQuestion;
+const endQuestion = require('../models/issue').endQuestion;
+const getUserById = require('../models/user').getUserById;
+
+const issue = (socket) => {
+  socket.on('action', (data) => {
+    const payload = data.data;
+    logger.debug('issue payload', { payload, action: data.type });
+    if (data.type === 'server/ADMIN_CREATE_ISSUE') {
+      addQuestion(payload)
+      .then((question) => {
+        logger.debug('Added new question. Broadcasting ...', { question: question.description });
+        broadcast(socket, 'OPEN_ISSUE', question, { action: 'open' });
+        return null;
+      }).catch((err) => {
+        logger.error('Adding new question failed.', { err });
+        emit(socket, 'issue', {}, {
+          error: 'Adding new question failed',
```

Comment:
Again, concerning errors: seems to me like the normal thing to do is have error be a boolean, and provide an Error object as the payload.